vendor/github.com/openebs/api/v2/pkg/apis/cstor/v1/cstorvolume.go

```diff
@@ -151,7 +151,7 @@ type ReplicaStatus struct {
 	InflightSync string `json:"inflightSync"`
 	// time since the replica connected to target
 	UpTime int `json:"upTime"`
-	// Quorum indicates wheather data wrtitten to the replica
+	// Quorum indicates whether data wrtitten to the replica
```

Comment:
Can we update vendor changes in  [API](https://github.com/openebs/api) repo, instead of updating the vendor files directly ?

Reply:
Removed vendor changes.